pkg/scaffold/controller/controller.go

```diff
@@ -120,8 +119,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 	{{ .Resource.Group}}{{ .Resource.Version }} "{{ .ResourcePackage }}/{{ .Resource.Group}}/{{ .Resource.Version }}"
+)
+
+var logf = log.Log.WithName("controller")
```

Comment:
we usually do the other way around -- `logf` is the package alias, and `log` is the name of the actually base logger.

Reply:
Yes, I used `log` before, but it conflicted with "the import log pkg" in test file (I assumed it should not to use the controller-runtime logger here, right?).
e.g. 
https://github.com/kubernetes-sigs/controller-tools/blob/master/test/pkg/controller/firstmate/firstmate_controller_suite_test.go#L20.

So one of them should change and I choose to change here to `logf`.
Is there a better way? Or a more appropriate name instead of `logf`? 
Thanks.

Reply:
@mengqiy 
Yes, I noticed it before and the test code of my pr has been generated by the script. 
If used `var log = ...`, still have this confilction, reason see my previous comments.

Thanks.

Reply:
It probably shouldn't be importing the standard "log" package -- just the controller runtime log package.

Reply:
@DirectXMan12 Yes, agree with you.
So what's your idea about "log" name confliction? Change `logf` to a more appropriate name?
Thanks.